cmd/occollector/app/builder/processor_builder.go

```diff
@@ -63,6 +67,30 @@ func NewJaegerThriftHTTPSenderCfg() *JaegerThriftHTTPSenderCfg {
 	return opts
 }
 
+// BatchingConfig contains configuration around the queueing batching.
+// It contains some advanced configurations, which should not be used
+// by a typical user, but are provided as advanced features to increase
+// scalability.
+type BatchingConfig struct {
+	// Enabled marks batching as enabled or not
+	Enabled bool `mapstructure:"enabled"`
```

Comment:
Perhaps `Enable` as `Enabled` implies a past tense and the equivalent of having already been applied.